scripts/script.js

```diff
@@ -65,24 +62,24 @@ export default function () {
  * @param {string} piece The piece CID string to fetch
  * @returns A K6 HTTP response from the BOOST_FETCH_URL (https://k6.io/docs/javascript-api/k6-http/response/)
  */
-function fetchFromBoost(piece) {
-  let response = get(`${__ENV.BOOST_FETCH_URL}${piece}`, {
+function fetchFromBoost (piece) {
```

Comment:
Is a space between the function name and the arguments really a standardjs styling? Also nitpicking here, but that doesn't seem standard practice. I'm starting to feel like if we want to adhere to a specific opinionated styling, then it needs to be baked into the project otherwise it's not going to be adhered to over time.

Reply:
yeah, https://standardjs.com/
this has been my style for years now and it's been adopted pretty widely thanks to being baked into standard.js

Reply:
> then it needs to be baked into the project

suggestions? currently doesn't have a Makefile or a package.json, which is kind of nice

it's not super critical that it be adhered to over time, someone can show up now and again and `standard --fix` to sort it out

Reply:
figured out the answer to that .. github actions, added a simple lint action to this PR